actionpack/test/dispatch/request_test.rb

```diff
@@ -1304,3 +1304,20 @@ class RequestFormData < BaseRequestTest
     assert !request.form_data?
   end
 end
+
+class EarlyHintsRequestTest < BaseRequestTest
+  def setup
+    super
+    @env["rack.early_hints"] = lambda { |links| links }
+    @request = stub_request
+  end
+
+  test "when early hints is set in the env link headers are sent" do
+    early_hints = @request.send_early_hints([{ link: "a_path.css", as: "stylesheet" }, { link: "a_path.js", as: "script" }])
+
+    assert_equal early_hints.first[:link], "a_path.css"
+    assert_equal early_hints.first[:as], "stylesheet"
+    assert_equal early_hints.last[:link], "a_path.js"
+    assert_equal early_hints.last[:as], "script"
```

Comment:
The argument order here should be reversed. It's expected, then actual.

Reply:
Fixed when the API was updated.